src/asqi/schemas.py

```diff
@@ -150,6 +150,45 @@ class LLMAPIConfig(SystemDefinition):
         description="Parameters specific to the LLM API system (e.g., base url, model name, API key and env file).",
     )
 
+# RAG API system
+
+class RAGAPIParams(BaseModel):
+    """Parameters for the RAG API systems."""
+
+    base_url: str = Field(
+        ...,
+        description="Base URL for the OpenAI-compatible API (e.g., 'http://localhost:4000/v1', 'https://api.openai.com/v1')",
+    )
+    model: str = Field(
+        ...,
+        description="System name to use as model name with the API",
+    )
+    user_group: Optional[str] = Field(
+        None,
+        description="Indicates the role or access tier under which the query should be executed, for example: 'admin', 'support-agent', 'regional-user'",
+    )
```

Comment:
I would suggest removing `user_group` - it should be a test-level parameter, not a system-level parameter:

1. System config describes the RAG endpoint (URL, auth, model) - it's infrastructure
2. `user_group` is a test parameter - you want to test DIFFERENT user groups against the SAME RAG system
3. RBAC testing requires multiple user groups - you can't test access control with a single hardcoded group